courses-rest/src/main/java/no/sikt/nva/fs/SecretFormatException.java

```diff
@@ -0,0 +1,16 @@
+package no.sikt.nva.fs;
+
+import java.net.HttpURLConnection;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+
+public class SecretFormatException extends ApiGatewayException {
+
+    public SecretFormatException(Exception exception, String message) {
+        super(exception, message);
+    }
+
+    @Override
+    protected Integer statusCode() {
+        return HttpURLConnection.HTTP_INTERNAL_ERROR;
```

Comment:
If it is to return an internal error, better to throw a RuntimeException, that way, we do not inform the client about the internals of our system through the stacktrace. Runtime exceptions are automatically mapped to internal errors by ApiGatewayHandler

Reply:
Ok, I will fix it.